src/main/java/uk/gov/companieshouse/certifiedcopies/orders/api/model/FilingHistoryDocument.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.companieshouse.certifiedcopies.orders.api.model;
+
+public class FilingHistoryDocument {
+
+    private String filingHistoryDate;
+
+    private String filingHistoryDescription;
+
+    private String filingHistoryId;
+
+    private String filingHistoryType;
```

Comment:
Could this possibly be an `enum`?

Reply:
It could possibly be. The specs don't say what the types are though. These fields will be populated in GCI-1209, perhaps in that story we can check if we can use an enum.